modules/local/epicore.nf

```diff
@@ -0,0 +1,55 @@
+process EPICORE {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::epicore=0.1.5"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/epicore:0.1.5--pyhdfd78af_0' :
+        'biocontainers/epicore:0.1.5--pyhdfd78af_0' }"
+
+    input:
+        tuple val(meta_fasta), path(fasta)
+        path(quantification_tsv)
```

Comment:
Rename to result_tsv.

Also I think result_tsv and general_stats files should be imported as a tuple to not mix up different files from different samples. So `tuple meta, path(result_tsv), path(general_stats)`

I think you need to adjust the summarize_result output such that a tuple of meta, result_tsv, general_stats is emitted. That channel can then be used as input for epicore